res/css/style.css

```diff
@@ -64,83 +64,6 @@ body,
   flex-shrink: 1;
 }
 
-.tabBarContainer {
```

Comment:
I think that's been moved already, you can probably remove this commit while rebasing and that will fix most of your conflicts.